weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/customWeaveType.types.ts

```diff
@@ -0,0 +1,48 @@
+export type CustomWeaveTypePayload<
+  T extends string = string,
+  FP extends {[filename: string]: string} = {[filename: string]: string}
+> = {
+  _type: 'CustomWeaveType';
+  weave_type: {
+    type: T;
+  };
+  files: FP;
+  load_op?: string | CustomWeaveTypePayload<'Op', {'obj.py': string}>;
+} & {[extra: string]: any};
+
+export const isCustomWeaveTypePayload = (
+  data: any
```

Comment:
can `any` include `undefined`? might want to relax the strictness of the `===` in that case